src/structurizr/view/view_set.py

```diff
@@ -321,12 +348,11 @@ def _find_component_view(self, view: ComponentView) -> Optional[ComponentView]:
                 return current_view
         return None
 
-    # TODO: dynamic view
-    # def find_dynamic_view(self, view: DynamicView) -> DynamicView:
-    #     for current_view in self.dynamic_views:
-    #         if view.key == current_view.key:
-    #             return current_view
-    #     return None
+    def _find_dynamic_view(self, view: DynamicView) -> Optional[DynamicView]:
+        for current_view in self.dynamic_views:
+            if view.key == current_view.key:
+                return current_view
+        return None
```

Comment:
```suggestion
        return next((current_view for current_view in self.dynamic_views if current_view.key == view.key), None)
```

Reply:
I like it, but when it runs through `black` then it turns into:
```python
        return next(
            (
                current_view
                for current_view in self.dynamic_views
                if current_view.key == view.key
            ),
            None,
        )
```

Is that still better than the original?

Reply:
Hmm, let's stick with the outer `for` loop but please remove the `return None`. It's superfluous.

Reply:
Want me to remove the `return None` from the other similar methods whilst I'm here?

Reply:
Yes, please. I did comment where I saw it but probably missed some 🙂